src/components/SingleHeroBanner.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faStar } from '@fortawesome/free-solid-svg-icons';
+import PropTypes from 'prop-types';
+
+
+export default class SingleHeroBanner extends React.Component {
+    static propTypes = {
+        hero: PropTypes.object,
+        showModal: PropTypes.func,
+    }
+
+    addToFavorites = (name, id, publisher, image) => {
+        const favoritesArray = JSON.parse(window.localStorage.getItem("favorites")) || [];
+        if (favoritesArray.filter(item => item.id === id).length > 0) {
+            this.props.showModal("You already liked this character")
+        } else {
+            const item = {
+                name,
+                id,
+                publisher,
+                image
+            }
+            favoritesArray.push(item);
+            window.localStorage.setItem("favorites", JSON.stringify(favoritesArray));
+            this.props.showModal(`${name} was added to your favorites`);
+        }
+    }
+
+    render() {
+        const { image, name, id } = this.props.hero
+        const { biography } = this.props.hero;
+        const publisher = biography.publisher;
+        const alterEgos = biography["alter-egos"];
+        const placeOfBirth = biography["place-of-birth"];
+        const fullName = biography["full-name"];
+
+        const renderAliases = () => {
+            const { aliases } = this.props.hero.biography;
+            if (aliases) {
```

Comment:
Consider something like:
```
if (aliases.length > 1) {
  [aliases.slice(0, aliases.length - 2).join(', '), aliases.slice(aliases.length - 2, aliases.length - 1)].join(' and ')
} else {
  ...aliases
}
```
I know that's kinda hard to read, but basically if the list larger than 1, take the last element and join the beginning elements with commas and append the last element with `and`.